code/modules/clothing/clothing.dm

```diff
@@ -38,7 +38,10 @@
 	var/cooldown = 0
 	var/species_disguise = null
 	var/magical = FALSE
+	/// Can we use dye to altar this piece of clothing? Currently used by the washing machine
 	var/dyeable = FALSE
+	/// The associated key to the dye registry GLOB list, used to transform/color this piece of clothing through dyeing
+	var/dyeing_key = ""
```

Comment:
Since we always use defines for these, I don't think we should specify them as strings.
```suggestion
	var/dyeing_key
```